src/ucp/proto/proto_am.inl

```diff
@@ -20,20 +20,33 @@
 
 #define UCP_STATUS_PENDING_SWITCH (UCS_ERR_LAST - 1)
 
-#define UCP_AM_BCOPY_HANDLE_STATUS(_multi, _status) \
+/**
+ * @brief Handle status from AM Bcopy operation.
+ *
+ * @param [in]     _multi       Flags whether this is multi AM Bcopy or not.
+ * @param [in/out] _status      Completion status of send operation.
+ * @param [in/out] _exit_action Operation that should be done to stop macro
+ *                              execution, if a send operation completed:
+ *                              - for multi AM Bcopy with switcing to an another
+ *                                lane for pending.
+ *                              - with lack of resource status.
```

Comment:
what is special about those 2 cases that they have exit action?
seems like a random choice of what case is handled by this macro vs. what is handler by the macro user

Reply:
yes, I agree that this is not so clear
The special cases are about situations when operation is not completed 
I will think how to re-write it